.github/workflows/ci.yml

```diff
@@ -21,7 +21,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macOS-latest]
-        rust: [stable, 1.40.0]
+        rust: [stable, 1.45.0]
```

Comment:
Isn't 1.42.0 sufficient? That is the first version where [`use proc_macro::TokenStream` works](https://blog.rust-lang.org/2020/03/12/Rust-1.42.html#use-proc_macrotokenstream-now-works).